packages/ui/src/common/components/EditableInputList/EditableInputList.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react'
+import { ChangeEvent, useState } from 'react'
+import styled from 'styled-components'
+
+import { ButtonBareGhost } from '@/common/components/buttons'
+import { TransactionButton } from '@/common/components/buttons/TransactionButton'
+import { InputComponent, InputNotification, InputText, ToggleCheckbox } from '@/common/components/forms'
+import { BinIcon } from '@/common/components/icons'
+import { PlusIcon } from '@/common/components/icons/PlusIcon'
+import { TextHuge, TextMedium } from '@/common/components/typography'
+
+export interface EditableInputListProps {
+  title?: string
+  buttonText: string
+}
```

Comment:
The component works well but unless I'm missing something, to make it usable it should be a [controlled components](https://reactjs.org/docs/forms.html#controlled-components) with a `value` and an `onChange` function, similarly to how the `ToggleCheckbox` is implemented